code/modules/surgery/mcomp_tendwounds.dm

```diff
@@ -0,0 +1,171 @@
+
+/datum/surgery_step/mstabilize_wounds
+	name = "stabilize wounds"
+	desc = "stabilize the wounds"
+	tools = SURGERY_TOOLS_MEDICOMP_STABILIZE_WOUND
+	time = 5 SECONDS
+
+/datum/surgery_step/mstabilize_wounds/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, tool_type, datum/surgery/surgery)
+	if(user == target)
+		user.visible_message(SPAN_NOTICE("[user] begins to stabilize wounds on their  body with \the [tool]."),
+		SPAN_HELPFUL("You begin to stabilize your wounds with \the [tool]"))
+	else
+		user.affected_message(target,
+		SPAN_HELPFUL("You begin to stabilize the wounds on [target]'s body with \the [tool]."),
+		SPAN_HELPFUL("[user] begins to stabilize the wounds on your body with \the [tool]."),
+		SPAN_NOTICE("[user] begins to stabilize the wounds on [target]'s body with \the [tool]."))
+
+/datum/surgery_step/mstabilize_wounds/success(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool, tool_type, datum/surgery/surgery)
+	target.heal_overall_damage(50,50)
+
+	if(isSpeciesYautja(target))
+		target.emote("click2")
+	else
+		target.emote("pain")
+
+	if(!target.getBruteLoss() && !target.getFireLoss())
+		if(user == target)
+			user.visible_message(SPAN_NOTICE("[user] finshes to stabilizing the wounds on their body with \the [tool]."),
+			SPAN_HELPFUL("You finished to stabilize your wounds with \the [tool]"))
+	else
+		user.affected_message(target,
+			SPAN_HELPFUL("You finish stabilizing [target]'s wounds with \the [tool]."),
+			SPAN_HELPFUL("[user] finished stabilizing your wounds with \the [tool]."),
+			SPAN_NOTICE("[user] finished treating [target]'s wounds with \the [tool]."))
+
+		log_interact(user, target, "[key_name(user)] stabilized some of [key_name(target)]'s wounds with \the [tool].")
+
+/datum/surgery_step/mstabilize_wounds/failure(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool, tool_type, datum/surgery/surgery)
+	log_interact(user, target, "[key_name(user)] failed to stabilize [key_name(target)]'s wounds with \the [tool], possibly ending [surgery].")
+	return FALSE
+
+/datum/surgery_step/mtend_wounds
+	name = "Tend Wounds"
+	desc = "tend the wounds"
+	tools = SURGERY_TOOLS_MEDICOMP_MEND_WOUND
+	time = 15 SECONDS
+
+/datum/surgery_step/mtend_wounds/extra_checks(mob/living/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, repeating, skipped)
+	var/obj/item/tool/surgery/healing_gun/gun = tool
+	if(!gun.loaded)
+		to_chat(user, SPAN_WARNING("You can't heal yourself without a capsule in the gun!"))
+		return FALSE
+	return TRUE
+
+/datum/surgery_step/mtend_wounds/preop(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool, tool_type, datum/surgery/surgery)
+	if(user == target)
+		user.visible_message(SPAN_NOTICE("[user] begins to treat the stabilized wounds on their  body with \the [tool]."),
+		SPAN_HELPFUL("You <b>begin to treat your stabilized wounds with \the [tool]"))
+		playsound(target,'sound/misc/heal_gun.ogg',25)
```

Comment:
put this outside the if, it's useless to have in both the if and the else if it's the same